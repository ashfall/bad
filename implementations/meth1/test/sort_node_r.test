#!/bin/sh

mkdir -p ${srcdir}/.test-tmp
rm -f ${srcdir}/.test-tmp/sort_node_r.out

${srcdir}/app/meth1_node_test_r \
  0.1 \
  ${srcdir}/test/in.s0000.e2000.recs \
  | grep "last:" \
  > ${srcdir}/.test-tmp/sort_node_r.out

diff \
  ${srcdir}/test/sort_node_r.out \
  ${srcdir}/.test-tmp/sort_node_r.out

