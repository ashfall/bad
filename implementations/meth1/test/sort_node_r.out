last: 185bac34ef8cf4170296
last: 307f50832d696b7b84d9
last: 495c3563e917b0eb19a0
last: 61a09ccf4da905b4b61d
last: 7a713557cc075c635fff
last: 92a73b02d8ada4afd8c8
last: b0795011b782d9fc1037
last: cbe026d025eff80cb94c
last: e4be9c1b01bc58e6a759
last: ffb4d5e8475ed345334b
